reader.cc

```diff
@@ -308,11 +315,24 @@ void InitDaemon(const std::vector<std::string>& files) {
 
       for (int i = 0; i < length;) {
         inotify_event* event = reinterpret_cast<inotify_event*>(&buffer[i]);
-        if (event->mask & IN_MODIFY) {  // If the event was a modify event
+        i += EVENT_SIZE + event->len;
+
+        // Must be a modify event
+        if (!(event->mask & IN_MODIFY))
+          continue;
```

Comment:
Can you add some curly braces to these? We would like to conform to the Google C++ style guide if possible.

Other than that, LGTM!

Reply:
Done